File: backend/src/main/java/org/jdk/project/dto/config/ImageDto.java
package org.jdk.project.dto.config;

import lombok.Builder;
import lombok.Getter;

@Getter
@Builder
public class ImageDto {
  private Long id;
  private String imgName;
  private String imgType;
  private String roomCode;
  private String deptName;
  private Integer width;
  private Integer height;
  private Integer enable;
  private String remark;
}

File: backend/src/main/kotlin/org/jdk/project/dto/config/ImageDto.kt
package org.jdk.project.dto.config

data class ImageDto(
    var id: Long? = null,
    var imgName: String? = null,
    var imgType: String? = null,
    var roomCode: String? = null,
    var deptName: String? = null,
    var width: Int? = null,
    var height: Int? = null,
    var enable: Int? = null,
    var remark: String? = null,
)
